adapter/internal/messaging/notification_listener.go

```diff
@@ -126,8 +127,10 @@ func processNotificationEvent(conf *config.Config, notification *msg.EventNotifi
 		handleApplicationEvents(decodedByte, eventType)
 	} else if strings.Contains(eventType, subscriptionEventType) {
 		handleSubscriptionEvents(decodedByte, eventType)
-	} else {
+	} else if strings.Contains(eventType, policyEventType) {
 		handlePolicyEvents(decodedByte, eventType)
+	} else {
+		// filter all other events including HEALTH_CHECK event
```

Comment:
We do not need an else block to keep a comment. Shall we remove that?

Reply:
resolved via : https://github.com/wso2/product-microgateway/pull/2731/commits/8d45d46298b6216131e2be9154d8cc1778eaebc5